Docker-compose.yml

```
@@ -0,0 +1,27 @@
+version: '3'
+services: 
+  app:
+    build: .
+    ports:
+      - "5000:5000"
+    command: bundle exec rails s -p 5000
+    depends_on:
+      - iis-sandbox
+      - v2-to-fhir
+    links:
+      - iis-sandbox
+      - v2-to-fhir
+    environment:
+      IIS_SANDBOX_USERNAME: mitre
+      IIS_SANDBOX_PASSWORD: mitre
+      IIS_SANDBOX_FACILITY_ID: MITRE Healthcare
+      IIS_SANDBOX_HOST: http://localhost:8081
+      V2_TO_FHIR_HOST: http://localhost:3000
+  iis-sandbox:
+    build: ./IIS-Sandbox
+    ports:
```

```suggestion
    expose:
      - "8080"
```

We can publish this port without exposing it on the host machine ([Docker-compose#expose](https://docs.docker.com/compose/compose-file/compose-file-v3/#expose))

---

I've found it helpful to have the IIS-Sandbox exposed on the host machine so that I can use the GUI to see what patients exist that I am able to query for. Is there a big downside to keeping this? Alternatively I could just keep this exposed in a separate branch to use for demos and testing.